docs/README.md

```diff
@@ -0,0 +1,54 @@
+# Catalyst Client v2 implementation examples
```

Comment:
v2 is not really a proper name, I think we are on the 10 version or something, sorry about the misleading branch name :joy:

Reply:
You're right, should I mention anything about a version upgrade or just removing the v2 is enough since from now on this is going to be the only way to implement these workflows?

Reply:
yes, I think we can assume this is the new default, we can create a new doc for upgrading if necessary